### PR TITLE
refactor: change to docusaurus

### PR DESCRIPTION
**Pull request by tac0turtle**
blocked on final release till we get logos and images from the marketing team

### CODE REVIEW

File: website/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
```
Comment (samricotta) [inline website/README.md:3]: Not sure if you wanna update this to not include the above

Comment (tac0turtle) [reply website/README.md:3]: i just left that readme as is since the system is using the README.md in cips directory

File: .github/CODEOWNERS
Comment (julienrbrt) [inline .github/CODEOWNERS:1]: this shouldn't be empty.

File: .gitignore
Comment (julienrbrt) [inline .gitignore:1]: we may want to keep vscode here, and move the one from website/.gitignore here.

**Review by julienrbrt — COMMENTED** [review]
image

Nothing is being shown / integrated in the website.

File: website/src/pages/markdown-page.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: Markdown page example
+---
+
+# Markdown page example
+
+You don't need React to write simple standalone pages.
```
Comment (samricotta) [inline website/src/pages/markdown-page.md]: this can also be removed

**Comment by tac0turtle** [comment]
> image
> 
> Nothing is being shown / integrated in the website.
Screenshot 2023-07-18 at 10 48 31


when i run it there is a something, not sure what route you are hitting?

**Comment by julienrbrt** [comment]
> when i run it there is a something, not sure what route you are hitting?

Right, this is because the command I ran ignored the error from `yarn build`.
I ran `yarn build; yarn serve` and `yarn build` gives:

```sh
[ERROR] Unable to build website for locale en.
[ERROR] Error: Docusaurus found broken links!

Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.

Exhaustive list of all broken links found:

- On source page path = /process:
   -> linking to ../.github/CODEOWNERS (resolved as: /.github/CODEOWNERS)
   -> linking to ./cip-1.md#cip-header-preamble (resolved as: /cip-1.md)

    at throwError (/home/julien/projects/cosmos/cips/website/node_modules/@docusaurus/logger/lib/index.js:76:11)
    at handleBrokenLinks (/home/julien/projects/cosmos/cips/website/node_modules/@docusaurus/core/lib/server/brokenLinks.js:153:47)
    at async buildLocale (/home/julien/projects/cosmos/cips/website/node_modules/@docusaurus/core/lib/commands/build.js:186:5)
    at async tryToBuildLocale (/home/julien/projects/cosmos/cips/website/node_modules/@docusaurus/core/lib/commands/build.js:41:20)
    at async mapAsyncSequential (/home/julien/projects/cosmos/cips/website/node_modules/@docusaurus/utils/lib/jsUtils.js:34:24)
    at async Command.build (/home/julien/projects/cosmos/cips/website/node_modules/@docusaurus/core/lib/commands/build.js:76:21)
```

So it fails on broken links, `yarn start` works fine but gives the broken link warning too.

**Comment by tac0turtle** [comment]
> > when i run it there is a something, not sure what route you are hitting?
> 
> Right, this is because the command I ran ignored the error from `yarn build`. I ran `yarn build; yarn serve` and `yarn build` gives:
> 
> ```shell
> [ERROR] Unable to build website for locale en.
> [ERROR] Error: Docusaurus found broken links!
> 
> Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
> Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.
> 
> Exhaustive list of all broken links found:
> 
> - On source page path = /process:
>    -> linking to ../.github/CODEOWNERS (resolved as: /.github/CODEOWNERS)
>    -> linking to ./cip-1.md#cip-header-preamble (resolved as: /cip-1.md)
> 
>     at throwError (/home/julien/projects/cosmos/cips/website/node_modules/@docusaurus/logger/lib/index.js:76:11)
>     at handleBrokenLinks (/home/julien/projects/cosmos/cips/website/node_modules/@docusaurus/core/lib/server/brokenLinks.js:153:47)
>     at async buildLocale (/home/julien/projects/cosmos/cips/website/node_modules/@docusaurus/core/lib/commands/build.js:186:5)
>     at async tryToBuildLocale (/home/julien/projects/cosmos/cips/website/node_modules/@docusaurus/core/lib/commands/build.js:41:20)
>     at async mapAsyncSequential (/home/julien/projects/cosmos/cips/website/node_modules/@docusaurus/utils/lib/jsUtils.js:34:24)
>     at async Command.build (/home/julien/projects/cosmos/cips/website/node_modules/@docusaurus/core/lib/commands/build.js:76:21)
> ```
> 
> So it fails on broken links, `yarn start` works fine but gives the broken link warning too.

fixed now, thanks

File: website/docusaurus.config.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+              {
+                html: `<a href="https://cosmos.network"><img src="/img/logo-bw.svg" alt="Cosmos Logo"></a>`,
+              },
+            ],
+          },
+          {
+            title: "Documentation",
+            items: [
+              {
+                label: "Cosmos Hub",
```
Comment (julienrbrt) [inline website/docusaurus.config.js:91]: We should add an SDK docs link.

File: website/docusaurus.config.js [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+        sidebar: {
+          autoCollapseCategories: true,
+        },
+      },
+      navbar: {
+        title: "Cosmos CIPs",
+        hideOnScroll: false,
+        logo: {
+          alt: "Cosmos CIPs Logo",
+          src: "img/logo-sdk.svg",
```
Comment (julienrbrt) [inline website/docusaurus.config.js:68]: I know the logo isn't set yet, but this should have at least another name.

File: website/static/img/favicon.ico
Comment (julienrbrt) [inline website/static/img/favicon.ico:1]: nit, use another favicon

**Review by julienrbrt — APPROVED** [review]
lgtm.

File: website/docusaurus.config.js (hunk 4 above)
Comment (tac0turtle) [reply website/docusaurus.config.js:68]: waiting on the marketing team

File: website/static/img/favicon.ico
Comment (tac0turtle) [reply website/static/img/favicon.ico:1]: marketing team

**Pull request merged**